cmd/minikube/cmd/ssh-host.go

```diff
@@ -67,7 +67,7 @@ func appendKnownHelper(nodeName string, appendKnown bool) {
 		}
 	}
 
-	scanArgs := []string{"-t", "rsa"}
+	scanArgs := []string{"-t", "ed25519"}
```

Comment:
We use this in too many places - add a constant so we can have the algorithm exactly once in the project.